packages/server-node/src/launch/socket-server-launcher.ts

```diff
@@ -30,11 +30,24 @@ export class SocketServerLauncher extends GLSPServerLauncher<net.TcpSocketConnec
 
     run(opts: net.TcpSocketConnectOpts): Promise<void> {
         const netServer = net.createServer(socket => this.createClientConnection(socket));
+
         netServer.listen(opts.port, opts.host);
-        this.logger.info(`The GLSP server is ready to accept new client requests on port: ${opts.port}`);
-        // Print a message to the output stream that indicates that the start is completed.
-        // This indicates to the client that the sever process is ready (in an embedded scenario).
-        console.log(this.startupCompleteMessage);
+        netServer.on('listening', () => {
+            const addressInfo = netServer.address();
+            // eslint-disable-next-line no-null/no-null
+            if (addressInfo === null) {
```

Comment:
You can  avoid the null check and eslint supression by using this:
```suggestion
            if (!addressInfo) {
```

Reply:
Thanks @tortmayr ! Changed in current commit